### DISCUSSION TITLE
Unified PCB design

### DISCUSSION THREAD

**Discussion by chris-010** [hrvach/deskhop#102] [Ideas]
By looking at the pcb i realized that both sides are nearly identical. What about a unified pcb design for both (a/b) sides, with the only difference of soldered isolator? So, there is just one small unified board with functionality for side-a with isolator soldered and the same board for side-b w/o isolator. Then tie the seperated boards together to interconnect the rx/tx signals and you are done. Please don't ask me about the advantages of that stragetry, I'm currently try to find some.

Here a draft of my idea, sorry for less details, but i hope you get the main idea of this cascaded design. 

![image](https://github.com/hrvach/deskhop/assets/10660568/aa55bb8d-8128-464c-a31f-b3a05a421864)

**Comment by chris-010** [comment]
By thinking about the advantages, i hopefully found one.
Maybe this could be the base design when Deskhop is able to work with more than 2 computers?

**Reply by ams1** [reply]
If we use a Pico W instead of a Pico and enable Bluetooth - see #100 -, using only one of your boards we could switch between 2 computers: a usb connection (and power) to computer 1 and a Bluetooth connection to computer 2.

We get only 1 x USB host though... If we could fit 2...

**Comment by hrvach** [comment]
This is a pretty cool concept, I was thinking of doing something similar with a future board revision to enable connecting 2 boxes to get 4 outputs.
